<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChuBuFangAnForm</class>
 <widget class="QWidget" name="ChuBuFangAnForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>初步方案生成</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>361</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>初步整治方案如下：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_generate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>440</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
